### PR TITLE
adding gaussian 16 for AVX and SSE2

### PR DESCRIPTION
**Pull request by shahzebsiddiqui**

### CODE REVIEW

File: easybuild/easyconfigs/g/gaussian/gaussian-16-AVX.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    'G%(version_major)sBASIS': '%(installdir)s/g%(version_major)s/basis',
+}
+
+modextrapaths = {
+    'GAUSS_EXEDIR': ['g%(version_major)s', 'g%(version_major)s/bsd'],
+    'PATH': ['g%(version_major)s/bsd', 'g%(version_major)s'],
+    'LD_LIBRARY_PATH': 'g%(version_major)s',
+}
+
+moduleclass = 'bio'
```
Comment (verdurin) [inline easybuild/easyconfigs/g/gaussian/gaussian-16-AVX.eb]: Why `bio` here?

Comment (verdurin) [inline easybuild/easyconfigs/g/gaussian/gaussian-16-AVX.eb]: Please also address the style problems reported by Travis:
```
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/g/gaussian/gaussian-16-AVX.eb:27:1: W191 indentation contains tabs
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/g/gaussian/gaussian-16-AVX.eb:33:1: E101 indentation contains mixed spaces and tabs
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/g/gaussian/gaussian-16-AVX.eb:40:121: E501 line too long (121 > 120 characters)
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/g/gaussian/gaussian-16-SSE2.eb:27:1: W191 indentation contains tabs
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/g/gaussian/gaussian-16-SSE2.eb:33:1: E101 indentation contains mixed spaces and tabs
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/g/gaussian/gaussian-16-SSE2.eb:40:121: E501 line too long (121 > 120 characters)
```

Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/g/gaussian/gaussian-16-AVX.eb]: so what moduleclass should it be?

**Comment by migueldiascosta** [comment]
@shahzebsiddiqui, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**